### PR TITLE
Jl merge protocols

### PR DESCRIPTION
**Pull request by jleonardw9**

### CODE REVIEW

File: lib/tasks/merge_protocols.rake [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    first_protocol.attributes.each do |attribute, value|
+      if (attribute != 'id') && (attribute != 'type') && (attribute != 'created_at') && (attribute != 'updated_at') && (attribute != 'deleted_at')
+        second_protocol_attributes = second_protocol.attributes
+        if value != second_protocol_attributes[attribute]
+          resolved_value = resolve_conflict(attribute, value, second_protocol_attributes[attribute])
+          first_protocol.assign_attributes(attribute.to_sym => resolved_value)
+        end
+      end
+    end
+  end
```
Comment (ghost) [inline lib/tasks/merge_protocols.rake]: I think you need an `else` here to quit the task.

File: lib/tasks/merge_protocols.rake [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  if first_protocol.valid?
+    first_protocol.save
+  else
+    puts "#" *20
+    puts first_protocol.errors.inspect
```
Comment (ghost) [inline lib/tasks/merge_protocols.rake]: If you use `raise` over `puts`, it'll quit the task, which is what you want to do here, I think.

**Review by jayhardee9 — COMMENTED** [review]
What about human_subjects_info, investigational_products_info, etc. child objects? Should those be transferred as well? I'm assuming the study_type_questions wouldn't move over. Do Protocols have Documents or Notes that need to be moved as well?

**Comment by jleonardw9** [comment]
@jayhardee9 Forgot about the notes, thanks! I transfer over the documents in line 104. I'll implement your other suggestions as well.

**Review by jayhardee9 — APPROVED** [review]

**Review by Stuart-Johnson — APPROVED** [review]

**Pull request merged**